portfolio/src/main/webapp/index.html

```diff
@@ -3,17 +3,210 @@
+        </div>
+      </section>
+    </header>
+
+    <main class="container">
+      <p class="scroll" id="about">Scroll down to find out more</p>
+        
+      <section class="my-stack" id="about">
+        <h2>About Me</h2>
+        <div class="row no-row">
+          <div class="col-md-6">
+            <img src="img/correctClass.jpg" class="img-margin small-screen" />
+            <img src="img/code_tint.jpg" class="img-margin" />
+          </div>
+          <div class="col about-me">
+            <h3 class="small-screen">my <br /> story</h3>
+            <p>
+             Hello, my name is Nateyana!
+            </p>
+          </div>
+        </div>
+      </section>   
+      
+      <div id="hello-div"></div>
+
+  
+      <!-- works -->
+      <section class="projects" id="works">
+        <h2>works</h2>
+
+        <div class="card-styles">
+
+          <div class="card">
+            <a href="https://github.com/nblake707/recipe-app">
+              <div class="card-image-1"></div>
+            </a>
+            <div class="card__content">
+              <p>Recipe App</p>
+            </div>
+          </div>
+
+          <div class="card">
+            <a href="https://github.com/nblake707/video-search-app">
+              <div class="card-image-2"></div>
+            </a>
+            <div class="card__content">
+              <p>React Search App</p>
+            </div>
+          </div>
+
+          <div class="card">
+            <a href="https://github.com/nblake707/SuperheroSightings">
+              <div class="card-image-3"></div>
+            </a>
+            <div class="card__content">
+              <p>superhero sightings</p>
+            </div>
+          </div>
+
+      </div>
+
+        <div class="card-styles">
+          <div class="card">
+            <a href="https://github.com/nblake707/VendingMachineWebApp">
+            <div class="card-image-4"></div> 
+          </a>
+            <div class="card__content">
+              <p>Vending Machine</p>
+            </div>
+          </div>
+
+          <div class="card">
+            <a href="https://github.com/nblake707/VendingMachineAjax">
+              <div class="card-image-5"></div>
+            </a>
+            <div class="card__content">
+              <p>Vending Machine Ajax</p>
+            </div>
+          </div>
+
+          <div class="card">
+            <a href="https://github.com/nblake707/survey-collection-app">
+              <div class="card-image-6"></div>
+           </a>
+            <div class="card__content">
+              <p>Email Survey App</p>
+            </div>
+          </div>
+
+          </div> 
+      </section>
+
+      <!-- contact -->
+      <section class="contact" id="contact">
+        <h2>contact</h2>
+          <div class="row form_div">
+            <div class="col-sm">
+              <p>Feel free to reach out!</p>
+              <!-- <p><strong>Email: <a>nblake707@gmail.com</a></strong></p> -->
```

Comment:
Don't we need to specify the contact?

Reply:
I'm using formspree for form submissions